src/pages/Volunteer.jsx

```diff
@@ -8,8 +8,11 @@ const Volunteer = () => {
     // either make Hero reusable somehow (and move it to components/global)
```

Comment:
Can delete all the comments, now that you figured out direction.

Reply:
removed.